baselines/dom.generated.d.ts

```diff
@@ -13009,10 +13009,14 @@ interface ElementDefinitionOptions {
     extends: string;
 }
 
+interface ElementCreationOptions {
+    is: string;
```

Comment:
It looks like `is` is optional as I read the spec

Reply:
Months later, but `is` is optional. Web IDL spec says a dictionary member is by default optional unless it has `required` keyword.

Reply:
Ah, you are right. I'll fix this PR.
https://heycam.github.io/webidl/#prod-DictionaryMember